<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoolWindow</class>
 <widget class="QWidget" name="PoolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>831</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayoutPool">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Saved packets</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableLeft">
      <column>
       <property name="text">
        <string>Packet</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Pool</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="buttonUpdate">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="buttonSend">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableRight">
        <column>
         <property name="text">
          <string>Packet</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Count</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Delay (msec)</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutDevice">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;Not detected&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonChangeDevice">
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
